transaction-retry/src/gas_price_increase.rs

```diff
@@ -4,8 +4,9 @@ use futures::stream::{Stream, StreamExt as _};
 // least 12.5%.
 const MIN_GAS_PRICE_INCREASE_FACTOR: f64 = 1.125 * (1.0 + f64::EPSILON);
 
-pub fn minimum_increase(gas_price: f64) -> f64 {
-    (gas_price * MIN_GAS_PRICE_INCREASE_FACTOR).ceil()
+/// The minimum gas price that allows a new transaction to replace an older one.
```

Comment:
Maybe we should add `#![deny(missing_docs)]` to the crate?

Reply:
I'm slightly opposed to it because it forces documenting some things that are obvious enough from the name that the documentation doesn't add anything. But  no strong preference either way.

Reply:
Let's maybe try it? I'm personally very bad at thinking about docs so having a linter yell at me might be a good thing.